### ISSUE TITLE
Hello! I got some trouble when I ran the "torch_fedavg_mnist_lr_custum_data_and_model_example.py"

### ISSUE THREAD

**Issue by dmndxld** [FedML-AI/FedML#446]
I want to use fedml to do some work. To custom my data and my model, I did this.

```import fedml
import torch
from fedml.data.MNIST.data_loader import download_mnist, load_partition_data_mnist

from fedml.simulation import SimulatorSingleProcess as Simulator
from data_loader import load_partition_data_fMoW

from resnet import ResNet18


def load_data(args):
    
    
    #fedml.logging.info("load_data. dataset_name = %s" % args.dataset)
        (
         train_data_num,
        test_data_num,
        train_data_global,
        test_data_global,
        train_data_local_num_dict,
        train_data_local_dict,
        test_data_local_dict,
        class_num,
    ) = load_partition_data_fMoW(
        dataset='fMoW',data_dir="/data/yc/leaf_new_data/fMoW/", partition_method="hetero",partition_alpha=0.1,client_number=100, batch_size=16
    )
    """
    For shallow NN or linear models, 
    we uniformly sample a fraction of clients each round (as the original FedAvg paper)
    """
    args.client_num_in_total = 100
    dataset = [
        train_data_num,
        test_data_num,
        train_data_global,
        test_data_global,
        train_data_local_num_dict,
        train_data_local_dict,
        test_data_local_dict,
        class_num,
    ]
    return dataset, class_num



net = ResNet18()


if __name__ == "__main__":
    # init FedML framework
    args = fedml.init()

    # init device
    device = fedml.device.get_device(args)

    # load data
    dataset, output_dim = load_data(args)



    # load model (the size of MNIST image is 28 x 28)
    #model = LogisticRegression(28 * 28, output_dim)
    model=net

    # start training
    simulator = Simulator(args, device, dataset, model)
    simulator.run()
```

The code could run, but print something I don't need, like this
![7e155a02d4a46aff97bbded2bbf043d](https://user-images.githubusercontent.com/46319943/182628762-7f35a838-bdb4-4ad3-b4f6-0ba9e9493315.jpg)
then get in a loop
![e62ecdfc1edf370ca8735b836bbf778](https://user-images.githubusercontent.com/46319943/182628855-47382b38-744f-4838-b672-361933f81307.jpg)

I want to get accuracy. I dont know where is the key to problem? Could you help me out? Thanks!

**Comment by dmndxld** [comment]
I just make a little different in fedml_config.yaml
`common_args:
  training_type: "simulation"
  random_seed: 0
  config_version: "release"
  mlops_api_key: c9356b9c4ce44363bb66366b210201
  mlops_project_name: alex-test-prj
  mlops_run_name: fedml_torch_fedavg_mnist_lr_2

data_args:
  dataset: "mnist"
  data_cache_dir: ~/fedml_data
  partition_method: "hetero"
  partition_alpha: 0.1

model_args:
  model: "resnet"

train_args:
  federated_optimizer: "FedAvg"
  client_id_list: "[]"
  client_num_in_total: 100
  client_num_per_round: 2
  comm_round: 10
  epochs: 1
  batch_size: 16
  client_optimizer: sgd
  learning_rate: 0.03
  weight_decay: 0.001

validation_args:
  frequency_of_the_test: 2

device_args:
  using_gpu: true
  gpu_id: 0

comm_args:
  backend: "sp"

tracking_args:
  enable_tracking: true
  log_file_dir: ./log
  enable_wandb: false
  wandb_key: ee0b5f53d949c84cee7decbe7a629e63fb2f8408
  wandb_entity: fedml-ai
  wandb_project: simulation
  run_name: fedml_torch_fedavg_mnist_lr
  using_mlops: false`

**Comment by alex-liang-kh** [comment]
You may upgrade your fedml pip to new version: pip install -U fedml

**Comment by dmndxld** [comment]
> You may upgrade your fedml pip to new version: pip install -U fedml

Thanks for your reply. But it didnt work. Now it does not print anything.
![image](https://user-images.githubusercontent.com/46319943/182804388-6b66c909-c7ce-4db0-8d23-be17f9993b0a.png)
Now its 4:44pm
